src/main/resources/application.conf

```diff
@@ -86,12 +86,19 @@ workspace {
   workspacesEntitiesCopyPath="/workspaces/entities/copy"
   submissionsPath="/workspaces/%s/%s/submissions"
   submissionsIdPath="/workspaces/%s/%s/submissions/%s"
+  overwriteGroupMembershipPath="/admin/groups/%s/members"
 }
 
 userprofile {
   authPrefix = "/api"
   setKey="/thurloe"
-  getAll="/thurloe/%s"
   get="/thurloe/%s/%s"
+  getAll="/thurloe/%s"
+  getQuery="/thurloe?"
   delete="/thurloe/%s/%s"
 }
+
+nih {
+  whitelistFile = "users.txt"
+  rawlsGroupName = "dbGapAuthorizedUsers"
```

Comment:
this conflicts with #184 but, with the addition of the filename, I like this PR's version better